CHANGES_REQUESTED

Blocking finding:
- `frontend/app/services/groupAudio.ts:382` introduces a strict TypeScript failure. `event` is typed as `any`, so `stream` is also `any`, and `stream.getAudioTracks().forEach((track) => ...)` leaves `track` as an implicit `any` under the repo's `noImplicitAny` setting. `npx tsc --noEmit` fails with:

```text
app/services/groupAudio.ts(382,40): error TS7006: Parameter 'track' implicitly has an 'any' type.
```

Glare trace review:
- Simultaneous offers `a123` / `b456`: both enter `have-local-offer`; `b456 > a123` yields and answers; `a123 < b456` ignores; answer reaches `a123`. No Round 2 deadlock remains.
- Normal single-offer path from `stable` still accepts and answers.
- Stable stale-offer path still accepts and answers; redundant but not a blocker.

Scope check: only the expected three files changed (`groupAudio.ts`, `intercom.tsx`, `SPEC_REGISTER.md`). No merge performed.

Verification run:
- `npm ci` in `frontend/` succeeded.
- `npx tsc --noEmit` failed as above.
- `npm run lint` could not run because `eslint` is not installed in the frontend dev dependencies (`sh: 1: eslint: not found`).

--- a/frontend/app/services/groupAudio.ts
+++ b/frontend/app/services/groupAudio.ts
@@ -379,7 +379,7 @@ export class GroupAudioManager {
       if (!stream) return;
       record.remoteStream = stream;
       this.callbacks.onRemoteStream?.(peer.userId, stream);
-      stream.getAudioTracks().forEach((track) => {
+      stream.getAudioTracks().forEach((track: MediaStreamTrack) => {
         this.callbacks.onSpeakingTrack?.(peer.userId, !!track.enabled);
       });
     };

